### ISSUE TITLE
Zillions of Chrome tabs opened

### ISSUE THREAD

**Issue by Vanuan** [testem/testem#785]
The bug in fireworm causes testem to detect false file modifications, resulting in a new browser instance launched for each file.

Please update testem as soon as this is merged and a new version of fireworm released:
https://github.com/airportyh/fireworm/pull/10

**Comment by davidgoli** [comment]
:+1:

**Comment by davidgoli** [comment]
More info here: https://github.com/testem/testem/issues/777

**Comment by Vanuan** [comment]
Yeah, could be related.

**Comment by johanneswuerbach** [comment]
This https://github.com/testem/testem/pull/786 should fix the issue. I plan to release a new version including this tomorrow.

**Comment by Vanuan** [comment]
@johanneswuerbach No, this is a different one. Timeout is a crazy workaround. I don't want to wait more just because you can't write asynchronous code properly. You should fix the underlying issue in the file watcher.

**Comment by johanneswuerbach** [comment]
Sorry, but I guess you are mistaken here. The filewatcher is just triggering flawed changed events, but testem shouldn't open more then one tab (the filewatcher should still be fixed, but as you can see this isn't my repo / not my control). The referenced PR fixes this tab issue and introduces a new functionality (start timeout errors).

**Comment by Vanuan** [comment]
I'm sorry if I was too harsh, but I've spent too much time hunting down that bug.

**Comment by johanneswuerbach** [comment]
`testem@1.3.0` is now released an should fix the tab issue. Rephrased this issue title as a reminder to bump fireworm ones the new version is released.

**Comment by Vanuan** [comment]
Can't reproduce multiple tabs issue in 1.3.0. Thanks!

**Comment by Vanuan** [comment]
What I usually do in cases when dependency author ain't responding is forking a dependency and releasing a custom version.

**Comment by Vanuan** [comment]
Another instance of that bug: `before_tests` command  it is triggered twice for one modification.

**Comment by Vanuan** [comment]
I've published a forked, fixed version to https://www.npmjs.com/package/vanuan-fireworm
